### DISCUSSION TITLE
IOSF Virtual Meet Up

### DISCUSSION THREAD

**Discussion by GouravSardana** [IndianOpenSourceFoundation/dynamic-cli#114] [Q&A]
Hey Everyone,

I hope everything is good. Can we have a virtual meet up next this Saturday (July 3) where we will discuss about what's going on and what need to be done in Dynamic CLI ?

Also need your suggestions to start a new project under IOSF for all of you. It would be mosty on Frontend or might be on other technologies (depends on suggestions).

@sakshatl and ME would be there for you. Sakshat is handling our Product end. 

Are you guys available on Saturday ?

cc @kaustubhgupta @pranavbaburaj @anshsrtv @guptaaastha @harshakhmk @haider000 @Ak-Shaw @lakshyajit165 @NamamiShanker @honeybhardwaj

**Comment by honeybhardwaj** [comment]
@GouravSardana yeah am available.

**Reply by GouravSardana** [reply]
Will send you the invite

**Comment by lainq** [comment]
That's awesome. btw, what is the project about 🤔 ?

**Comment by NamamiShanker** [comment]
That's great! Maybe we can also discuss refactoring Dynamic CLI. Unfortunately, I will be out of the station but I will try to join.

**Comment by ghost** [comment]
Hello Everyone,
Try to be there on Saturday 3rd July. We will be mostly talking about some
new projects that we can take up, how to grow our community and also about
participation in open source contests and events like - hacktober, GSoC
etc. Would love to hear your thoughts about the organization as a whole.

*Looking forward to seeing you all :)*

PS: if any of you is not able to be there, that's fine we will keep the
session recorded and also document our facts and share them onto slack and
telegram channels

Cheers,
Sakshat

**Reply by guptaaastha** [reply]
hey, I couldn't attend due to some last minute work on other front. I'd like to get access to the meet recording. Please send it over at aasthagupta.071@gmail.com thanks 😄 
@sakshatl @GouravSardana

**Reply by GouravSardana** [reply]
Sure. Will share the recording tomorrow or day after tomorrow

**Reply by guptaaastha** [reply]
thanks @GouravSardana , that's great!

**Reply by guptaaastha** [reply]
@GouravSardana just wanted to remind you of this!
thanks

**Reply by GouravSardana** [reply]
I know i know doing some editing. Actually it was not a meet recording. It was screen recording. So i have to cut 10-20 sec in between.

Actually meet removed the record button. So just did screen recording on the last moment.

**Reply by guptaaastha** [reply]
No worries at all ! take your time to fix the meetup recording. I'll be waiting for it 😄

**Comment by harshakhmk** [comment]
Sounds exciting, I am willing to join

**Comment by guptaaastha** [comment]
Hi all,
I'll be available to join the meetup 😄, It'd be great if @GouravSardana @sakshatl  you could tell at what time is the meetup planned.

**Comment by kaustubhgupta** [comment]
@GouravSardana Great! Pl send a calendar invite for the same

**Comment by GouravSardana** [comment]
For timing - You guys can give suggestions 4PM ?

@kaustubhgupta Yes I'll send calendar invite, once time got confimed.

**Reply by honeybhardwaj** [reply]
yep saturday 4 works for me

**Reply by lainq** [reply]
Yes, I guess it works for me. I will make sure to attend 👍🏾

**Reply by guptaaastha** [reply]
works for me as well

**Comment by GouravSardana** [comment]
I have sent a calendar invite to your emails. Please do check it out and acknowledge/accept the invite. If anyone of you has received, please share your email id here or email me at ping@iosf.in

**Comment by ghost** [comment]
Hello everyone,

Thanks for attending the session today. You really made it a great one and
w/ all those fresh ideas popping in we all are motivated to bring the
products to life.

Also, for the ones who couldn't make it to the session we will be sharing
the  link to the recording with you once it is up.

Once again a big thanks, looking forward to more brainstorming sessions
like these 🎉

**Comment by GouravSardana** [comment]
Those who haven't attended the session. No worries, I'll share the recording via email.

**Comment by GouravSardana** [comment]
Sorry for the delay guys. My laptop took much time to render it even it was a small edit. 

Here is the recording - https://youtu.be/V5MfCgSVl8g

If you like it. Do like the video. Comment some suggestions on video improvement and Do subscribe to our channel 💯 


cc @kaustubhgupta @pranavbaburaj @anshsrtv @guptaaastha @harshakhmk @haider000 @Ak-Shaw @lakshyajit165 @NamamiShanker @honeybhardwaj

**Reply by guptaaastha** [reply]
great! thanks @GouravSardana

**Reply by NamamiShanker** [reply]
Thanks @GouravSardana
